### ISSUE TITLE
Trampoline allocation underflow in SKSEVR

### ISSUE THREAD

**Issue by joseph-maher** [ianpatt/skse64#14]
Hey fellas, since there is no SKSEVR repo I'm going to post this here

SKSEVR on linux is still suffering from a trampoline underflow that was fixed quite some time ago in SKSE64

The exact commit that sorted it in SKSE64 is [https://github.com/ianpatt/skse64/commit/4a1e121](here)

Cheers

**Comment by jotoho** [comment]
Related to #1

**Comment by joseph-maher** [comment]
yes, I am aware this issue has been fixed for SKSE64, but the latest build for SKSEVR predates this fix, so it is still broken

**Comment by jotoho** [comment]
> yes, I am aware this issue has been fixed for SKSE64, but the latest build for SKSEVR predates this fix, so it is still broken

I never claimed otherwise

**Comment by joseph-maher** [comment]
and yet its interesting to note that you did not claim likewise either

**Comment by jotoho** [comment]
> and yet its interesting to note that you did not claim likewise either

My intention was to create a link between those two issues for searchability. Who knows, maybe it'll help someone out in the future.
It was never a message meant for you specifically.
I am not affiliated with SKSE and uninterested in dealing with someone apparently in a foul enough mood to lash out at others. Unsubscribing from this issue. Do not bother me again.
